---
title: Quality Control for RNA Sequencing
layout: abstract_details
permalink: /abstracts/31/
published: true
isPublic_b: true
tabTitle_txt: Abstract Details

posternumber_ti: 31

categories_list: 
  - category_txt: Adaptive Oncology

keywords_txt: >-
  RNA; RNASeq; quality control; QC
  
authors_list:
  - author_txt: Iain Bancarz
  - author_txt: Savo Lazic
  - author_txt: Lawrence Heisler
  - author_txt: Nicholas Khuu

affiliations_list:
  - affiliation_txt: OICR (IB, SL, LH)
  - affiliation_txt: UHN Research (NK)

presenting-author_txt: Iain Bancarz
presenting-author-email_txt: ibancarz@oicr.on.ca
presenting-author-title_txt: OICR, Bioinformatics Software Developer

text_txt: |-
  High-throughput sequencing of RNA (RNASeq) provides valuable insight into the transcriptome. It has substantial and growing applications in cancer research, particularly with the rise of single-cell RNASeq technologies. To enable maximum benefit from RNASeq, a robust system of quality control (RNASeq-QC) is needed -- for evaluation of sequencing read quality, and exclusion of probable erroneous data from further analysis. RNASeq-QC is based on metrics which provide quantitative indicators of data quality, accompanied by visualisation of metric values with graphics and reports. The RNASeq-QC domain presents its own challenges, distinct from those of QC for genomic DNA sequencing.
  
  Several RNASeq-QC software packages are available, each implementing a selection of metrics and visualisations. The Genome Sequence Informatics group at OICR uses elements of multiple packages, implemented as an automated workflow. We survey the current state of development in RNASeq-QC; present the output of RNASeq-QC on OICR production data; and discuss which QC outputs are most informative for assessment of RNASeq data quality.

---
