---
title: TGL Bioinformatics pipelines for analysing and reporting high quality data from whole exome and whole transcriptome sequencing platforms
layout: abstract_details
permalink: /abstracts/68/
published: true
isPublic_b: true
tabTitle_txt: Abstract Details

posternumber_ti: 68

categories_list: 
  - category_txt: Adaptive Oncology

keywords_txt: >-
  bioinformatics pipelines; ngs; exomes; rnaseq; qc

authors_list:
  - author_txt: Prisni Rath
    reference: 1
  - author_txt: Jonathon Torchia
    reference: 1
  - author_txt: Alka Benawra
    reference: 1
  - author_txt: Alexander Fortuna
    reference: 1
  - author_txt: Alberto J. LeÃ³n
    reference: 1
  - author_txt: Michael Laszloffy
    reference: 2
  - author_txt: Xuemei Luo
    reference: 2
  - author_txt: Yogi Sundaravadanam
    reference: 2
  - author_txt: Peter Ruzanov
    reference: 2
  - author_txt: Heather Armstrong
    reference: 2
  - author_txt: Dillan Cooke
    reference: 2
  - author_txt: Iain Bancarz
    reference: 2
  - author_txt: Andre Masella
    reference: 2
  - author_txt: Alexis Varsava
    reference: 2
  - author_txt: Richard Jovelin
    reference: 2
  - author_txt: Kayla Marsh
    reference: 1
  - author_txt: Jenna Eagles
    reference: 1
  - author_txt: Madhuran Thiagarajah
    reference: 1
  - author_txt: Lawrence Heisler
    reference: 2
  - author_txt: Morgan Taschuk
    reference: 2
  - author_txt: Lars G T Jorgensen
    reference: 2
  - author_txt: Dax Torti
    reference: 1
  - author_txt: Trevor J Pugh
    reference: 1, 3

affiliations_list:
  - affiliation_txt: PM-OICR Translational Genomics Laboratory
    reference: 1
  - affiliation_txt: Genome Sequence Informatics, Ontario Institute for Cancer Research
    reference: 2
  - affiliation_txt: Princess Margaret Cancer Centre, University Health Network
    reference: 3

presenting-author_txt: Prisni Rath
presenting-author-email_txt: Prisni.Rath@oicr.on.ca
presenting-author-title_txt: PM-OICR TGL, Ontario Institute of Cancer Research; Bioinformatician

text_txt: >-
  Rapid interpretation and reporting of clinical sequencing results presents a unique informatics challenge in certified clinical laboratories.  The Translational Genomics Laboratory (TGL) has profiled over 1,660 patients, 637 transcriptomes, 920 exomes, and 1,280 methylation arrays from more than 40 translational cancer research studies since 2017, thus necessitating the need for rapid and reproducible analysis. As part of this effort, we have expanded OICR's suite of automated bioinformatic pipelines to analyze sequencing data from whole exome (WES) and whole transcriptome  (RNA-Seq) sequencing platforms. Our pipelines utilize sample information captured in the MISO lab information system to execute pipelines via the Niassa analysis workflow framework. TGL employs a quality control dashboard based on the RShiny framework, for fast and efficient visualization of pre-analytic and deep sequencing results at single sample and project level resolution.  Our automated pipelines generate somatic and germline variants, copy number profiles, gene expression outliers, gene fusions and immunological profiles. These results are rapidly disseminated to collaborative networks using cBioPortal [http://cbioportal.ca](http://cbioportal.ca).  Our pipelines and infrastructure provides an end-to-end solution that ensures consistent, high-quality data is rapidly available for patient-level genome interpretation and reporting.

---
