---
title: Quantifying the Varying Selective Processes of Pre-Tumor and Tumor Evolution from High-Coverage Sequence Data
layout: abstract_details
permalink: /abstracts/22/
published: true
isPublic_b: true
tabTitle_txt: Abstract Details

posternumber_ti: 22

categories_list: 
  - category_txt: Adaptive Oncology

keywords_txt: >-
  Cancer progression; Evolutionary processes; Approximate Bayesian Computation
  
authors_list:
  - author_txt: Armande Ang Houle
    reference: 1, 2
  - author_txt: Kimberly Skead
    reference: 1, 2
  - author_txt: Boxi Lin
    reference: 1, 3
  - author_txt: Pamela Mehanna
    reference: 1 
  - author_txt: Mawusse Agbessi
    reference: 1
  - author_txt: Vanessa Bruat
    reference: 1
  - author_txt: Paul Boutros
    reference: 1, 4
  - author_txt: Stephen Wright
    reference: 1, 5
  - author_txt: David Soave
    reference: 1, 6 
  - author_txt: Lincoln Stein
    reference: 1, 2 
  - author_txt: Philip Awadalla
    reference: 1, 2

affiliations_list:
  - affiliation_txt: Computational Biology, Ontario Institute for Cancer Research
    reference: 1
  - affiliation_txt: Department of Molecular Genetics, Faculty of Medicine, University of Toronto
    reference: 2
  - affiliation_txt: Department of Statistics and Actuarial Science, Faculty of Mathematics, University of Waterloo
    reference: 3
  - affiliation_txt: Department of Medical Biophysics, Faculty of Medicine, University of Toronto
    reference: 4
  - affiliation_txt: Department of Ecology and Evolutionary Biology, Faculty of Art and Sciences, University of Toronto
    reference: 5
  - affiliation_txt: Department of Mathematics, Faculty of Sciences, Laurier University
    reference: 6

presenting-author_txt: Armande Ang Houle
presenting-author-email_txt: armande.anghoule@oicr.on.ca
presenting-author-title_txt: PhD Candidate

text_txt: >-
  The inference of cancer evolutionary processes greatly relies on the accurate characterization of somatic mutations. Selective sweeps may hinder the detection of negative selection within cancer cell populations. Our ability to capture these evolutionary signatures is dependent on sample size, which we infer using the read depth of somatic mutations to quantify the allelic frequency of a mutation within each population of cancer cells. Average sequencing depths are often insufficient to detect mutations found at low variant allele frequency within a population of cancer cells. Sequencing at higher depths allows for a more complete picture of the full evolutionary spectrum. Here, we identify evolutionary processes in 226 cancer and pre-cancer samples sequenced with a minimal coverage of 100X from 4 cancer types. We capture non-neutral evolutionary processes previously undetected in tumors, as well as varying rates of beneficial and deleterious mutations accumulating across patients using an Approximate Bayesian Computation framework. Additionally, we infer different evolutionary processes when ignoring the contribution of deleterious mutations. Similarly, inferred evolutionary processes strikingly differ based on the thresholds of minimal detected variant allelic frequencies. Our work demonstrates the heterogeneity of cancer evolutionary processes, and outlines the possibility of inaccurate inferences of cancer evolutionary processes caused by low sequencing depth.

---
