---
title: Driver Discovery in Localized Prostate Cancer
layout: abstract_details
permalink: /abstracts/11/
published: true
isPublic_b: true
tabTitle_txt: Abstract Details

posternumber_ti: 11 

categories_list: 
  - category_txt: Adaptive Oncology

keywords_txt: >-
  Prostate Cancer; Driver Discovery; Whole Genome Sequencing; 
  
authors_list:
  - author_txt: Helen Zhu on behalf of the Canadian Prostate Cancer Genome Network and the Australian Prostate Cancer Genome Project
  

affiliations_list:
  - affiliation_txt: Ontario Institute for Cancer Research
  - affiliation_txt: Department of Medical Biophysics, University of Toronto
  - affiliation_txt: Australian Prostate Cancer Research Centre
  - affiliation_txt: Department of Surgery, University of Melbourne
  - affiliation_txt: Department of Human Genetics, University of California, Los Angeles.

presenting-author_txt: Helen Zhu
presenting-author-email_txt: helen.zhu@oicr.on.ca
presenting-author-title_txt: OICR, Graduate Student

text_txt: |-
  Cancer is driven by the accumulation of somatic mutations. A small number of driver mutations confer a survival advantage to the tumour, while the majority are harmless passenger mutations. Therefore, discriminating between driver and passenger mutations requires strong evidence of functional impact and selection. Previously, driver discovery has focused on mutational impact in protein coding genes using whole exome sequencing datasets but as whole genome sequencing is becoming more prevalent, new tools are required for the exploration of the vast noncoding genome.
  
  ActiveDriverWGS was developed in our lab for the analysis of cancer whole genomes and the identification of recurrent single nucleotide variations and small insertions and deletions in protein coding genes and functional noncoding regulatory elements. Here, ActiveDriverWGS was applied to a collection of hundreds of primary prostate tumour whole genomes to find known coding driver mutations in genes such as SPOP, TP53 and PTEN and novel noncoding driver mutations in active enhancers, promoters and lncRNAs. We complemented our analysis with a comprehensive examination of recurrent genomic rearrangements in the nuclear genome and recurrent somatic single nucleotide variations in the mitochondrial genome. We conducted an integrative analysis on the impact of different classes of somatic aberrations using a novel method ActivePathways, revealing pathways affected by multiple types of somatic mutations.


---
