---
title: Exploiting population cohorts to interrogate genomic and transcriptomic factors associated with exceptional blood aging
layout: abstract_details
permalink: /abstracts/65/
published: true
isPublic_b: true
tabTitle_txt: Abstract Details

posternumber_ti: 65

categories_list: 
  - category_txt: Adaptive Oncology

keywords_txt: >-
  Aging; pleiotropy; populationn; cohort

authors_list:
  - author_txt: Elyssa Bader
  - author_txt: Marie-Julie Fave
  - author_txt: Mawusse Agbessi
  - author_txt: Elias Gbeha
  - author_txt: Vanessa Bruat
  - author_txt: Philip Awadalla

affiliations_list:
  - affiliation_txt: OICR, Dept. of Molecular Genetics University of Toronto, Montreal Heart Institute

presenting-author_txt: Elyssa Bader
presenting-author-email_txt: elyssa.bader@oicr.on.ca
presenting-author-title_txt: PhD candidate (OICR & Dept of Molecular Genetics, University of Toronto)

text_txt: >-
  While the effects of aging on blood cells have been well characterized at the cellular level, the underlying genomic architecture remains elusive. We hypothesize that pleiotropic variants can increase risk to develop multiple age-related conditions, and protective forms of these variants may underlie the genetic basis of healthy aging in blood. We use large population cohorts, the Canadian Partnership for Tomorrow Project (CPTP, n= ~320K) and the UK Biobank (UKBB; n = ~500K) to identify genetic determinants of aging in blood. We characterize blood aging using a risk score developed from complete blood count (CBC) data and find that healthy blood agers are less likely to be diagnosed with cardiovascular disease and leukemia. To identify pleiotropic variants associated with blood aging, we perform a phenome wide association study (PheWAS) in the UKBB with 65 649 candidate SNPs and over 500 traits. We identified thousands of significant associations, however many are likely false positives due to the LD structure between SNPs. To identify causal pleiotropic SNPs , we fine mapped each LD block containing two or more candidate SNPs for every trait tested. Current efforts aim to differentiate biological pleiotropy from mediated pleiotropy due to correlation between traits. Our results will provide insight into the maintenance of hematopoiesis over time. More broadly, our methods can be applied to identify underlying pleiotropy for other correlated traits, including cancers.

---
