---
title: The Transcriptomic and Proteogenomic Landscape of Curable Prostate Cancer
layout: abstract_details
permalink: /abstracts/27/
published: true
isPublic_b: true
tabTitle_txt: Abstract Details

posternumber_ti: 27

categories_list: 
  - category_txt: Adaptive Oncology

keywords_txt: >-
  integrative; transcriptome; protate cancer; biomarker
  
authors_list:
  - author_txt: Julie Livingstone
    reference: 1 
  - author_txt: Vincent Huang
    reference: 1
  - author_txt: Ankit Sinha
    reference: 2 
  - author_txt: Takafumi N. Yamaguchi
    reference: 1 
  - author_txt: Lawrence E. Heisler
    reference: 1 
  - author_txt: Yu-Jia Shiah
    reference: 1 
  - author_txt: Fouad Yousif
    reference: 1 
  - author_txt: Michael Fraser
    reference: 1
  - author_txt: Theodorus Van Der Kwast
    reference: 2 
  - author_txt: Hansen H. He
    reference: 2
  - author_txt: Robert G. Bristow
    reference: 2, 3 
  - author_txt: Thomas Kislinger
    reference: 2 
  - author_txt: Paul C. Boutros
    reference: 1, 4

affiliations_list:
  - affiliation_txt: Ontario Institute for Cancer Research
    reference: 1
  - affiliation_txt: Princess Margaret Cancer Centre
    reference: 2
  - affiliation_txt: Manchester Cancer Research Centre
    reference: 3
  - affiliation_txt: UCLA
    reference: 4

presenting-author_txt: Julie Livingstone
presenting-author-email_txt: julie.livingstone@oicr.on.ca
presenting-author-title_txt: Ontario Institute for Cancer Research, Toronto, ON, Bioinformatician

text_txt: |-
  Background: Prostate cancer remains the most common non-skin malignancy in men world-wide. Surprisingly, integration of the genome, epigenome, transcriptome and proteome remains relatively unexplored. 
  
  Methods: To fill this gap, we first performed ultra-deep non-poly-A RNA-Seq on 144 localized prostate samples and compared these transcriptomic profiles with somatic mutational features. Next, we performed an exhaustive integrative analysis of 76 localized prostate tumours using whole-genome sequencing, methylome and cis-regulatory element profiling, RNA-Sequencing and shotgun proteomic profiling. 
  
  Results: The prostate cancer transcriptome is defined by subgroups that differ from DNA-based subgroups.¬†The correlation between RNA and protein varied based on protein abundance and overall was poor (median: 0.21). Copy number aberrations had both cis and trans effects on RNA and protein abundance. Mutual information (MI) between data types varied, where combinations containing CNAs showed low MI and combinations with H3K27Ac showed high MI. Random forest classification models predicting biochemical relapse, revealed that proteomic features are significantly more informative than genomic, epigenomic or transcriptomic ones. 
  
  Conclusions: Globally, different biomolecules only weakly correlate and accordingly, protein abundance predicts disease aggressivity better than DNA or RNA features, and combinations of data-types improve biomarker accuracy.

---
